spring-ai-core/src/main/java/org/springframework/ai/chat/client/DefaultChatClientBuilder.java

```diff
@@ -71,6 +71,10 @@ public ChatClient build() {
 		return new DefaultChatClient(this.defaultRequest);
 	}
 
+	public Builder clone() {
```

Comment:
good catch

Reply:
As discussed, we figured the mystery :) The Builder has the "clone()" method and the Client APIs the "mutate()" method.